RELEASE_TEST_LOG.md

```diff
@@ -0,0 +1,12 @@
+# Holochain Release Log
+
+This file documents results of release tests as described in the [Holochain release process](RELEASE.md).
+
+## 2024-12-17: v0.4.0-rc.2
+
+- 3 nodes started a new network, peer discovery succeeded on all nodes within 5 minutes.
+- Sending signals to peers succeeded with close to 100 % reliability.
```

Comment:
I believe it was 100% reliability, right?

Reply:
I must have remembered that wrong.